springwolf-plugins/springwolf-spring-cloud-rabbit-binder-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/SpringwolfAmqpController.java

```diff
@@ -0,0 +1,53 @@
+package io.github.stavshamir.springwolf.asyncapi;
+
+import com.asyncapi.v2.model.channel.ChannelItem;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.github.stavshamir.springwolf.asyncapi.types.channel.operation.message.Message;
+import io.github.stavshamir.springwolf.producer.SpringwolfAmqpProducer;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Map;
+
+@Slf4j
+@RestController
+@RequestMapping("/springwolf/amqp")
```

Comment:
All names should change to fit the new plugin

Reply:
we should chat in discord.  I'd like to know more about the protocol designation and how it relates to this endpoint address